javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -14,19 +14,201 @@
 
 package build.bazel.tests.integration;
 
+import org.hamcrest.Description;
+import org.hamcrest.SelfDescribing;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+import static org.hamcrest.core.Is.is;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 import static com.google.common.truth.Truth.assertThat;
 import static org.junit.Assert.assertEquals;
 
-import org.junit.Test;
-
-/** {@link BazelBaseTestCase}Test */
+/**
+ * {@link BazelBaseTestCase}Test
+ */
+//suppress since same parameter value is ok for tests readability, tests should encapsulate and not hide
+@SuppressWarnings("SameParameterValue")
 public final class BazelBaseTestCaseTest extends BazelBaseTestCase {
 
-  @Test
-  public void testVersion() throws Exception {
```

Comment:
The style is not google java style :) I can do a pass with the formatter at the end of the review if you prefer.

Reply:
no, I'll do it. 
If I'm going to contribute code here I should setup my system :)

Reply:
done, I ran it on both java files.
Funny enough it changed some of your own code :)